xcmailr-webapp/src/main/java/controllers/AdminHandler.java

```diff
@@ -416,9 +416,10 @@ public Result addDomain(Context context, @Param("domainName") String domainName)
      *            the context of this request
      * @return
      */
-    public Result showEmailStatistics(Context context, @Param("dayPage") int dayPage, @Param("weekPage") int weekPage,
-                                      @Param("sortDailyList") String sortDailyList,
-                                      @Param("sortWeeklyList") String sortWeeklyList)
+    public Result showEmailStatistics(Context context, @Param("dayPage") Optional<Integer> dayPage,
+                                      @Param("weekPage") Optional<Integer> weekPage,
+                                      @Param("sortDailyList") Optional<String> sortDailyList,
+                                      @Param("sortWeeklyList") Optional<String> sortWeeklyList)
```

Comment:
The now optional parameters are never used. The UI does not send them either. Remove them?

Reply:
I made those parameters optional as sensible defaults can be used in case they are not given/valid. As pagination and sorting for statistics is currently not supported by UI but should be (or is planned to be at least).

Reply:
Removed unused parameters.